apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js

```diff
@@ -0,0 +1,226 @@
+									error={errors.experience}
+									cols={50}
+									rows={5}
+								/>
+							</atoms.Box>
+							<atoms.Box flexDirection='column'>
+								<Field
+									as={organisms.OpenResponse}
+									label='who do you think your idea is helpful to?'
+									placeholder='Describe your audience, including their demographic information, technology experience, why they would be interested in your idea, etc.'
+									id='targetAudience'
+									name='targetAudience'
+									cols={50}
+									rows={5}
+								/>
+							</atoms.Box>
+							<atoms.Box flexDirection='column'>
+								<Field
+									required
+									as={organisms.OpenResponse}
+									label='What Features would your Product have?&nbsp;'
+									placeholder='A list of possible features your product could have.'
+									id='features'
+									name='features'
+									touched={touched['features']}
+									error={errors.features}
+									cols={50}
+									rows={5}
+								/>
+							</atoms.Box>
+							<Field
+								as={organisms.OpenResponse}
+								label='Anything else you would like to share with us?'
+								placeholder='Want to share something else not listed above?'
+								id='extraInfo'
+								name='extraInfo'
+								cols={50}
+								rows={5}
+							/>
+							<Field
+								as={organisms.FormField}
+								label='Do you have a catchy tagline for this idea submission?'
+								placeholder='Your Tagline'
+								id='tagline'
+								name='tagline'
+							/>
+
+							<atoms.Box flexDirection='row' alignItems= 'flex-end' justifyContent= 'space-between'>
+								<Field as="select" name="involveLevel" style={{ fontSize: '1rem', padding: '0.5rem', width: '33rem' }}>
+									<option value="one">I don’t want to be involved after submitting</option>
+									<option value="two">I want to “own” this idea to help with workshopping and designing until it become a project</option>
+									<option value="three">I want to “own” this idea and also be part of the development/design team when it becomes a project</option>
+									<option value="four">I want to “own” this idea and also believe have the skills necessary to apply as the product leadto make it a successful project</option>
+								</Field>
+
+								<Popballoon
+									content="As an “idea owner” you’ll own the idea and be in charge of refine and update the information on the workshipping page."
+								/>
+							</atoms.Box>
+
+							<atoms.Typography type='p' style={{ marginTop: '0.5rem', }}>
+								After submitting your idea will be reviewed and enter the workshopping stage!
+							</atoms.Typography>
+
+
+							<atoms.Box style={{ fontSize: '1rem', }}>
+								<atoms.Checkbox
+									label='I have read and agree to the <a>Terms and Conditions</a>.'
+									disabled={false}
+									required
+								/>
+							</atoms.Box>
+
+							<atoms.Box style={{ justifyContent: 'flex-end' }}>
+								{formButton == "submit" ? (
```

Comment:
@alicefate it would make the code a lot more readable if we assigned the button outside of the `return` statement here, or extracted this button to its own component.

Reply:
You mean like this?
{formButton == "submit" ? ( <submissonButton />) : (<editionButton />)}